ci/Jenkinsfile

```diff
@@ -0,0 +1,5 @@
+node(label: 'linux') {
+  sh 'npm run docker-build'
```

Comment:
Do we have npm on `$PATH` here? If not I need to inline docker commands instead of relying on npm script.

Reply:
We could have but makes things more fragile. Neither yarn or npm handles concurrent builds very well, and npm doesn't even run concurrently very well so if we can skip using npm in the Jenkinsfile itself, it would be better as we can run multiple builds on one machine instead of one build per machine as now.